packages/editor/src/components/post-taxonomies/flat-term-selector.js

```diff
@@ -259,22 +326,55 @@ class FlatTermSelector extends Component {
 			_x( 'Remove %s', 'term' ),
 			singularName
 		);
+		const tagsButtonDescription = _x(
+			'Choose from the most used tags.',
+			'term'
+		);
 
 		return (
-			<FormTokenField
-				value={ selectedTerms }
-				suggestions={ termNames }
-				onChange={ this.onChange }
-				onInputChange={ this.searchTerms }
-				maxSuggestions={ MAX_TERMS_SUGGESTIONS }
-				disabled={ loading }
-				label={ newTermLabel }
-				messages={ {
-					added: termAddedLabel,
-					removed: termRemovedLabel,
-					remove: removeTermLabel,
-				} }
-			/>
+			<div>
+				<FormTokenField
+					value={ selectedTerms }
+					suggestions={ termNames }
+					onChange={ this.onChange }
+					onInputChange={ this.searchTerms }
+					maxSuggestions={ MAX_TERMS_SUGGESTIONS }
+					disabled={ loading }
+					label={ newTermLabel }
+					messages={ {
+						added: termAddedLabel,
+						removed: termRemovedLabel,
+						remove: removeTermLabel,
+					} }
+				/>
+				<div className="editor-post-taxonomies__flat-terms-related-list-wrapper">
+					<Button
+						className="is-link"
+						onClick={ () => this.toggleRelatedTerms() }
+						aria-expanded={ ! this.state.areRelatedTermsHidden }
+					>
+						{ tagsButtonDescription }
+					</Button>
+					{ this.state.areRelatedTermsHidden === false ? (
```

Comment:
I wouldn't use a ternary. Wouldn't the following be simpler? 
`{ ! this.state.areRelatedTermsHidden && ( <ul ....`